lib/rev-api/models/order_request.rb

```diff
@@ -68,7 +68,6 @@ class Payment < ApiSerializable
 
     # @param type [String] payment method
     def initialize(type)
-      warn "[DEPRECATION] `Payment` option is now optional, since it will always default to AccountBalance"
```

Comment:
Because it's noisy and useless. Whoever was using Payment should already punted it. Comment on the Payment still says deprecated: https://github.com/revdotcom/rev-ruby-sdk/pull/10/files#diff-47a4491cce92ee69e0156612533c6ddaR58